natspec/AllowlistV1.md

```diff
@@ -35,8 +35,23 @@ Sets the allowlisting status for one or more accounts
 
 | Name | Type | Description |
 |---|---|---|
-| _accounts | address[] | Accounts with statuses to edit |
-| _permissions | uint256[] | Allowlist permissions for each account, in the same order as _accounts |
+| _accounts | address[] | Accounts with statuses to be set to allow |
+
+### deny
+
+```solidity
+function deny(address[] _accounts) external nonpayable
```

Comment:
note that the `setDenierPermissions` is called `deny` here. Let's sync and then make a decision on the naming.

Reply:
is this autogenerated ? I guess this needs to be re-run if so ?

Reply:
Generated again